tests/testthat/test-estimate_KM.R

```diff
@@ -380,36 +380,28 @@ testthat::test_that("T6.4 The function adds strata labels equal to the strata na
 
 testthat::context("estimate_KM - T7. The function call supports traceability")
 
-testthat::test_that("T7.1 The function updates call$data when magrittr pipe is used", {
-
-  ## survival package
-  survobj_survival <- adtte %>%
-    survival::survfit(survival::Surv(AVAL, 1-CNSR) ~ SEX, data = .) %>%
-    survival::survfit0(start.time = 0)
-  call_survival <- as.list(survobj_survival[["call"]])
-
-  ## survival package
-  survobj_visR <- adtte %>%
+testthat::test_that("T7.1 The function updates .$data_name when magrittr pipe is used", {
```

Comment:
we should do this too with adtte[1:100,] or with double pipe adtte %>% mutate() %>%

Reply:
I added a test with `adtte[1:100,]`